services/value-store/projects/src/mappings/project.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  project: {
+    properties: {
+      budget: {
+        properties: {
+          funding_area: {
+            type: 'keyword',
+          },
+        },
+      },
+      computed_key: {
+        type: 'keyword',
+      },
+      coordinators: {
+        properties: {
+          country: {
+            type: 'keyword',
+          },
+        },
+      },
+      ec_priorities: {
+        type: 'keyword',
+      },
+      producer_id: {
+        type: 'keyword',
+      },
+      programme_name: {
+        type: 'keyword',
+      },
+      project_locations: {
+        properties: {
+          location: {
+            type: 'geo_point',
```

Comment:
Should be `geo_shape` https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-shape.html

Reply:
I'd suggest this change to happen in EUBFR-98 where we ensure the transform is also updated?